packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/general-information/general-information.html

```diff
@@ -170,7 +170,7 @@
                 class="oui-dropdown-option mb-1"
                 role="menuitem"
                 data-ng-if="$ctrl.bindings.numberOfIPBlocks.total"
-                data-ng-click="$ctrl.onAssociateIpBlock()"
+                data-ng-href="{{:: $ctrl.onAssociateIpBlock() }}"
```

Comment:
if duable, it's better to rename the variable to:
```suggestion
                data-ng-href="{{:: $ctrl.associateIpBlockLink }}"
```

Reply:
I renamed this fonction.